lib/cli/src/Cardano/CLI.hs

```diff
@@ -1605,10 +1689,10 @@ withLogging =
 withLoggingNamed
     :: LoggerName
     -> [LogOutput]
-    -> ((CM.Configuration, Trace IO Text) -> IO a)
+    -> ((Switchboard Text, (CM.Configuration, Trace IO Text)) -> IO a)
```

Comment:
It seems you changed this, but also currently ignore it?

Reply:
It is used here https://github.com/input-output-hk/cardano-wallet/pull/2322/files#diff-e12cf35dac52af31ad103700daedc7aea51fe3d2b10dbc97842e14e8c66bd8e9R299 and needed to start the snap server (ekg)